src/backend/core/api/viewsets.py

```diff
@@ -886,3 +889,20 @@ def perform_create(self, serializer):
         invitation.document.email_invitation(
             language, invitation.email, invitation.role, self.request.user
         )
+
+class ConfigView(views.APIView):
+    """API ViewSet for sharing some public settings."""
+
+    permission_classes = [AllowAny]
+
+    def get(self, request):
+        """
+        GET /api/v1.0/config/
+            Return a dictionary of public settings.
+        """
+        array_settings = ["SENTRY_DSN", "DJANGO_CONFIGURATION"]
```

Comment:
A great place to expose settings.LANGUAGE _CODE and settings.LANGUAGES so that the frontend starts with the same choice of languages and default language as the backend!
Any other settings that are duplicated between frontend and backend and that we could harmonize here?

Reply:
I separated the backend in this PR: https://github.com/numerique-gouv/impress/pull/425